MovieCatalogCrawler/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
+spring.datasource.username=asd
+spring.datasource.password=asd
+spring.h2.console.enabled=true
+
+
+hibernate.dialect=org.hibernate.dialect.H2Dialect
+hibernate.hbm2ddl.auto=create
```

Comment:
Any difference here if it is create or create-drop?

Reply:
It's create by default. Doesn't need to be changed.